content/installation.html

```diff
@@ -37,14 +37,14 @@ <h2>Prerequisites</h2>
 <h2><a name="marketplace"/>Option 1: Install from Eclipse Marketplace Client</h2>
 
 <p>
-  <a href="http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=264" title="Drag to your running Eclipse workspace to install EclEmma Java Code Coverage"> 
+  <a href="https://marketplace.eclipse.org/marketplace-client-intro?mpc_install=264" title="Drag to your running Eclipse workspace to install EclEmma Java Code Coverage"> 

```

Comment:
This change adds a trailing whitespace. Also IMO it is slightly unrelated to PR topic, so I'll split out this change of HTTP on HTTPS into a separate commit.